src/plugins/Plugin.js

```diff
@@ -2,6 +2,15 @@ const yo = require('yo-yo')
 // const nanoraf = require('nanoraf')
 
 /**
+ * Check if an object is a DOM element. Duck-typing based on `nodeType`.
+ *
+ * @param {*} obj
+ */
+function isDOMElement (obj) {
+  return obj && typeof obj === 'object' && obj.nodeType === Node.ELEMENT_NODE
+}
+
+/**
  * Boilerplate that all Plugins share - and should not be used
```

Comment:
Maybe we should move this function to a shared `helpers` folder. Like we currently have `Utils`, but I’d like to refactor that and have them as individual modules there.